src/helpers/dataFilter.tsx

```diff
@@ -0,0 +1,22 @@
+import { getCoordinates, getWeatherData, getAirQualityData} from './apiCalls';
```

Comment:
The fetch call we wrote to get the details for the cards on Home could live here too I think! When we merge in, we can move it over.